src/vs/base/browser/ui/selectBox/selectBox.css

```diff
@@ -6,4 +6,63 @@
 .monaco-workbench .select-box {
 	width: 100%;
 	height: 20px;
-}
\ No newline at end of file
+}
+
+/* Require .monaco-shell for ContextView dropdown */
+
+.monaco-shell .select-dropdown-container  {
+	display: none;
+}
+
+.monaco-shell .select-dropdown-container.visible  {
+	display: flex;
+	flex-direction: column;
+	text-align: left;
+	width: 1px;
+	overflow: hidden !important;
+}
+
+.monaco-shell .select-dropdown-container .selectbox-dropdown-list-container {
+	flex: 0 0 auto;
+	align-self: flex-start;
+	padding-bottom: 1px;
+	padding-top: 1px;
+	padding-left: 1px;
+	padding-right: 1px;
+	width: 100%;
+	overflow: hidden !important;
+	-webkit-box-sizing:	border-box;
+	-o-box-sizing:		border-box;
+	-moz-box-sizing:	border-box;
+	-ms-box-sizing:		border-box;
+	box-sizing:			border-box;
+}
+
+.monaco-shell.hc-black .select-dropdown-container .selectbox-dropdown-list-container {
+	padding-bottom: 4px !important;
```

Comment:
@cleidigh not a big fan of using `!important` in CSS unless there is no other way. Is this really needed?

Reply:
Done.